docs/reference/signer-config.md

```diff
@@ -1,60 +1,100 @@
 <!--[metadata]>
 +++
-title = "Signer Configuration File"
-description = "Specifies the configuration file for Notary Signer"
-keywords = ["docker, notary, notary-signer, configuration"]
+title = "Signer Configuration"
+description = "Configuring the notary client, server and signer."
+keywords = ["docker, notary, notary-client, notary-server, notary server, notary-signer, notary signer"]
 [menu.main]
-parent="mn_notary"
-weight=90
+parent="mn_notary_config"
 +++
 <![end-metadata]-->
 
-# Notary Signer Configuration File
+
+# Notary signer configuration file
+
+This document is for power users of the [notary client](../advanced_usage.md),
```

Comment:
Shouldn't reference client. Client only users are completely unaware of the signer.